packages/model-viewer/src/utilities/fetch-script.ts

```diff
@@ -0,0 +1,10 @@
+export function fetchScript(src: string): Promise<Event> {
+    return new Promise((resolve, reject) => {
+        const script = document.createElement('script');
+        document.body.appendChild(script);
+        script.onload = resolve;
+        script.onerror = reject;
+        script.async = true;
+        script.src = src;
+    });
+}
```

Comment:
These  two new  files are pretty tiny (and would require the license header); can we just put these in CachingGLTFLoader instead?